OpenQA/Isotovideo/Utils.pm

```diff
@@ -20,7 +20,7 @@ use Mojo::URL;
 use Exporter 'import';
 use File::Spec;
 use Cwd;
-use bmwqemu 'diag';
+use bmwqemu;
```

Comment:
Since bmwqemu has two default exports, it might be better to do
`use bwmqemu ();` in order to import nothing at all.

Reply:
Ah, because `fileContent` and `save_vars` are exported otherwise even if they're unused, good point!

Reply:
We never seem to make use of those actually. Maybe they should just not be exported?

Reply:
good idea. I also cannot see it used implicitly in os-autoinst-distri-opensuse